examples/example.csv

```diff
@@ -0,0 +1,3 @@
+_id,_full_text,FID
```

Comment:
Now, I see that I didn't change the path correctly. The file is needed, but the instructions now say:

    cp examples/aircan-example-1.csv ~/airflow/

which points to the wrong path. I'll commit a new change to this PR for that. :+1: